app/models/project.rb

```diff
@@ -36,6 +36,9 @@ class Project < ApplicationRecord
   scope :completed, -> { where.not(completed_at: nil) }
   scope :open, -> { where(completed_at: nil) }
 
+  scope :assigned_to_caseworker, ->(user) { where(assigned_to: user).or(where(caseworker: user)) }
```

Comment:
I think this is right in the context of this work, but it does make me want to know if regional delivery officers want to see projects where they are assigned to the regainal delivery officer role.

Reply:
Actually, doesn't the next line do this? 

`scope :assigned_to_regional_delivery_officer, ->(user) { where(assigned_to: user).or(where(regional_delivery_officer: user)) }

Reply:
Yeah, don't worry - this was just a thought that popped out!